server/db/postgres.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var pg = require('pg');
+var Q = require('q');
+var camelize = require('underscore.string').camelize;
+var underscored = require('underscore.string').underscored;
+
+var config = require('../config');
+var log = require('../logger')('server.db.postgres');
+
+var dbParams = {
+  user: config.get('db.postgres.user'),
+  password: config.get('db.postgres.password'),
+  host: config.get('db.postgres.host'),
+  port: config.get('db.postgres.port'),
+  database: config.get('db.postgres.database'),
+  ssl: config.get('db.postgres.ssl')
+};
+
+// primitive DB object
+var postgres = {
+  timeout: config.get('db.postgres.queryTimeout'),
+  // postgres is case-insensitive, so we store keys as underscored there,
+  // and transform back to camelCase when returning results here.
+  // individual queries must be sure to use the underscored versions.
+  camelize: function(rows) {
+    var outputRows = [];
+    var output, k;
+    rows.forEach(function(row) {
+      output = {};
+      Object.keys(row).forEach(function(k) {
+        output[camelize(k)] = row[k];
+      });
+      outputRows.push(output);
+    });
+    return outputRows;
+  },
+  // query postgres; returns a promise
+  //
+  // query := a prepared query string
```

Comment:
I guess these docs are nice, but it seems weird to only insert them in one spot.